### ISSUE TITLE
[RF Protocol][NVM] Verify current behaviour & adapt NVM management

### ISSUE THREAD

**Issue by ralcaide** [Sinapse-Energia/EC-LC-NEMA#9]
The legacy code implements already the Sinapse RF Protocol. 
The device have a concrete behavior depending some configuration parameters saved in the NVM. For example, a device can be a master and then will repeat the frames that arrives with transit bit = 1 or can be not master and then process only the frames with sent to it.
All the protocol was already available in the legacy code and should works fine but is necessary to check the needed values saved in the NVM are well saved, that could be read and also that could be written.

To check that everything works right it is necessary to test (and probably modify the NVM management) the setting master and then verify the behaviour.

The NVM management is explained here: #3 but we think that should be adapted to this new device. What is the best way to save these necessary parameters in the NVM (Flash). Using a page? other method?.

Test TODO:

1. Set the device as master using the command "Configure Radio" explained in the Wiki:

Configure Radio | 0x16 | P1 = Radio configuration | 0x01 |   | 0C 16 00 00 01 00 00 02 RC 01 06 CS -- 0B 01 00 00 02 00 00 01 01 06 CS
-- | -- | -- | -- | -- | --

The but RC should have the master = 1 like 1000111. This RC means master enabled and time of re-emisions = 700ms

2. Send a frame to other EC (other address) and verify that the EC master repeat the frame
3. Set the device as not master
4. Send a frame to other EC and verify that the current EC is not re-transmitting the frame.

TODO:

- Perform test
- Improve NVM management
- Perform test again until the device works as expected

**Comment by ralcaide** [comment]
@m-owais , everything clear with this task? Please provide feedback

**Comment by m-owais** [comment]
So far old code complies with above requirements.

**Comment by m-owais** [comment]
So far old code complies with above requirements.

**Comment by m-owais** [comment]
There are alot of unused defines in deficionesGlobales.h. Also there is unused code in tramas.c, can you help me identify which part will not be used in future so i can remove it. It makes the maintenance easier.

**Comment by ralcaide** [comment]
Yes, our developer will check the code and will let you know. Thanks

**Comment by ralcaide** [comment]
@francisjjp , please could you take a look?

**Comment by sinapseenergia** [comment]
All code from line 449 to 589.

**Comment by ralcaide** [comment]
@m-owais , what is the status of this task?. Is already done and to be tested?

**Comment by m-owais** [comment]
All extra code has been removed. There are some more candidates:

COMANDO_EXT_HABILITAR_ACK_DIFUSION is not defined well. Is it needed ?
COMANDO_LEER_TTL
COMANDO_ESCRIBIR_TTL
COMANDO_PETICION_TEST
COMANDO_ENABLE_IMALIVE
COMANDO_READ_EEPROM

are some commands which are implmented but i am not sure if they are needed.

**Comment by ralcaide** [comment]
Ok, we will verify.
@francisjjp , could you please check it?
